helm/adelphi/templates/cassandra-mgmt-dockerfile-configmap.yaml

```diff
@@ -19,7 +19,7 @@ data:
     COPY management-api-server/pom.xml ./management-api-server/pom.xml
     # this duplicates work done in the next steps, but this should provide
     # a solid cache layer that only gets reset on pom.xml changes
-    RUN mvn -q -ff -T 1C install && rm -rf target
+    RUN mvn -q -ff -T 1C install -DskipTests && rm -rf target
```

Comment:
This speeds up the execution for me (we shouldn't be re-testing the MGMT API in our end either).